<ui version="4.0" >
 <class>DBTableViewClass</class>
 <widget class="QWidget" name="DBTableViewClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="focusPolicy" >
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle" >
   <string>Table View</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="RoseApple.qrc" >
    <normaloff>:/RoseApple/Resources/set.png</normaloff>:/RoseApple/Resources/set.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" >
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Fiter:</string>
     </property>
     <property name="buddy" >
      <cstring>filterEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QComboBox" name="usrCmbbox" >
     <property name="editable" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="2" >
    <widget class="QLineEdit" name="filterEdit" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3" >
    <widget class="QSplitter" name="splitter" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget" >
      <layout class="QVBoxLayout" name="verticalLayout" >
       <item>
        <widget class="QFrame" name="frame_2" >
         <property name="frameShape" >
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow" >
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2" >
          <item>
           <widget class="QRadioButton" name="tableRdo" >
            <property name="text" >
             <string>Table</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="viewRdo" >
            <property name="text" >
             <string>View</string>
            </property>
            <property name="checked" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="sysTableRdo" >
            <property name="text" >
             <string>System Table</string>
            </property>
            <property name="checked" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="allRdo" >
            <property name="text" >
             <string>All</string>
            </property>
            <property name="checked" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QListView" name="listView" >
         <property name="contextMenuPolicy" >
          <enum>Qt::CustomContextMenu</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QTabWidget" name="tabWidget" >
      <property name="focusPolicy" >
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="currentIndex" >
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab" >
       <attribute name="title" >
        <string>Columns</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout" >
        <item row="0" column="0" >
         <widget class="QLabel" name="tableNameLbl" />
        </item>
        <item row="1" column="0" >
         <widget class="QTableView" name="columnTable" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="selectionBehavior" >
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="dataTab" >
       <attribute name="title" >
        <string>Data</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3" >
        <item row="0" column="0" >
         <widget class="QFrame" name="frame" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="frameShape" >
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow" >
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout" >
           <property name="spacing" >
            <number>2</number>
           </property>
           <property name="leftMargin" >
            <number>2</number>
           </property>
           <property name="topMargin" >
            <number>0</number>
           </property>
           <property name="rightMargin" >
            <number>2</number>
           </property>
           <property name="bottomMargin" >
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="dataLabel" />
           </item>
           <item>
            <widget class="QToolButton" name="filterApplyBtn" >
             <property name="icon" >
              <iconset resource="RoseApple.qrc" >
               <normaloff>:/RoseApple/Resources/play.png</normaloff>:/RoseApple/Resources/play.png</iconset>
             </property>
             <property name="checkable" >
              <bool>true</bool>
             </property>
             <property name="checked" >
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="filterBtn" >
             <property name="icon" >
              <iconset resource="RoseApple.qrc" >
               <normaloff>:/RoseApple/Resources/filter.png</normaloff>:/RoseApple/Resources/filter.png</iconset>
             </property>
             <property name="checkable" >
              <bool>true</bool>
             </property>
             <property name="checked" >
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QSplitter" name="vSplitter" >
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="handleWidth" >
           <number>4</number>
          </property>
          <widget class="QTextEdit" name="qryFilterEdit" >
           <property name="focusPolicy" >
            <enum>Qt::StrongFocus</enum>
           </property>
          </widget>
          <widget class="GTableView" name="dataTable" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="selectionBehavior" >
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="ddlTab" >
       <attribute name="title" >
        <string>DDL</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4" >
        <item row="0" column="0" >
         <widget class="QTextBrowser" name="ddlBrowser" />
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>GTableView</class>
   <extends>QTableView</extends>
   <header>GTableView.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>filterEdit</tabstop>
  <tabstop>listView</tabstop>
  <tabstop>columnTable</tabstop>
 </tabstops>
 <resources>
  <include location="RoseApple.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>filterBtn</sender>
   <signal>clicked(bool)</signal>
   <receiver>qryFilterEdit</receiver>
   <slot>setShown(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>497</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel" >
     <x>463</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>filterBtn</sender>
   <signal>toggled(bool)</signal>
   <receiver>filterApplyBtn</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>485</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel" >
     <x>508</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
